config.py

```
@@ -0,0 +1,381 @@
+def get_cache_dir() -> Path:
+    """Gibt das Cache-Verzeichnis zurück."""
+    cache_dir = get_app_data_dir() / "cache"
+    cache_dir.mkdir(exist_ok=True)
+    return cache_dir
+
+def is_admin() -> bool:
+    """Prüft, ob die App mit Admin-Rechten läuft."""
+    try:
+        import ctypes
+        return ctypes.windll.shell32.IsUserAnAdmin()
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)